src/screens/docs/components/markdown.js

```diff
@@ -62,7 +62,8 @@ class Markdown extends React.Component {
     });
 
     md.use(markdownItTocAndAnchor, {
-      anchorLinkSymbol: "",
+      anchorLinkSymbol: "heading anchor link",
```

Comment:
Ok. I like this thought, but this phrase ends up getting repeated, and I don’t think it’s useful. 

Here’s the document outline now: 
<img width="648" alt="screen shot 2016-09-30 at 1 26 28 pm" src="https://cloud.githubusercontent.com/assets/768965/19005833/8e042e96-8711-11e6-9cba-0d1f4cf537c8.png">

Ideally, we’d have the opposite of `visually-hidden` something like `screenreader-hidden` since the links are available in the sidebar. Or we’d add something like `aria-hidden="true"` if we had access to the markup. 

What do you think?